styleguides/html.md

```diff
@@ -0,0 +1,38 @@
+_This styleguide has been adapted from
+https://google.github.io/styleguide/htmlcssguide.xml._
+
+-------------------------------------------------------------------------------
+
+# HTML Styleguide
+
+This is a collection of HTML guidelines that Hack Club workshops should follow
```

Comment:
What if we changed it to the following?

> This is a collection of HTML guidelines for Hack Club workshops

I think that leaves enough ambiguity to have occasional deviations, but it's
also strongly worded enough to make it clear that these are the default
guidelines for workshops.